services/gundeck/test/integration/TestSetup.hs

```diff
@@ -0,0 +1,22 @@
+module TestSetup where
```

Comment:
this commit is already on develop.  i think we've both decided to do that at the same time.  should we all report in a lot more detail about things we're currently working on?  this is already on develop, not sure whether you did it before or after it got merged...

Reply:
good change, though!  :)

Reply:
Yeah I'm noticing some conflicts in the rebase 😖  I'll try to merge our brilliance together :)